src/util/merkleblock.rs

```diff
@@ -532,81 +532,94 @@ mod tests {
     use crate::util::merkleblock::{MerkleBlock, PartialMerkleTree};
     use crate::Block;
 
-    #[test]
-    fn pmt_tests() {
+    /// accepts `pmt_test_$num`
+    fn pmt_test_from_name(name: &str) {
+        pmt_test(name[9..].parse().unwrap())
+    }
+
+    macro_rules! pmt_tests {
+    ($($name:ident),* $(,)?) => {
+         $(
+        #[test]
+        fn $name() {
+            pmt_test_from_name(stringify!($name));
+        }
```

Comment:
nit: maybe some more identation

Reply:
We have `rustfmt` now.

Reply:
I applied `rustfmt` on modified code, unless i missed something?

Reply:
I know, I explained to @RCasatta that changing indentation is not possible.